wicket-core/src/main/java/org/apache/wicket/model/StringResourceModel.java

```diff
@@ -275,110 +276,70 @@ public void setObject(String object)
 	 * not be obtained from resource bundles that are held relative to a particular component or
 	 * page. However, for application that use only global resources then this parameter may be
 	 * null.
-	 * <p>
-	 * The model parameter is also optional and only needs to be supplied if value substitutions are
-	 * to take place on either the resource key or the actual resource strings.
-	 * <p>
-	 * The parameters parameter is also optional and is used for substitutions.
 	 * 
 	 * @param resourceKey
 	 *            The resource key for this string resource
 	 * @param component
 	 *            The component that the resource is relative to
-	 * @param model
-	 *            The model to use for property substitutions
-	 * @param parameters
-	 *            The parameters to substitute using a Java MessageFormat object
 	 */
-	public StringResourceModel(final String resourceKey, final Component component,
-		final IModel<?> model, final Object... parameters)
+	public StringResourceModel(final String resourceKey, final Component component)
```

Comment:
To make the migration easier I think it would be good to have one more constructor: 
`public StringResourceModel(final String resourceKey, final Component component, final IModel<?> model)`.